components/splash/cards/whyFBCards.js

```diff
@@ -37,7 +37,6 @@ const Card = ({ icon, text, heading }) => (
       as='h3'
       marginBottom='1rem'
       align='left'
-      lineGap='1'
```

Comment:
this was throwing an error bc it's not a valid prop/css thing ? I removed it and noticed no change.